### PR TITLE
small tweak to the colour of our red

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Darken the colour of the red based on feedback from readers. Makes the byline legible on immersive cards. 

Before
picture 1043
After
picture 1042

### CODE REVIEW

**Review by zeftilldeath — APPROVED** [review]

**Review by NataliaLKB — CHANGES_REQUESTED** [review]
I will approve once Gustavo approves 😄

**Comment by blongden73** [comment]
@NataliaLKB These have already been approved by Gus, its based on a conversation that we had this morning.

**Comment by GustavoGava** [comment]
Approved!

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**
